src/core/util.zig

```diff
@@ -0,0 +1,17 @@
+const std = @import("std");
+
+const log = std.log.scoped(.mach);
+
+// Instrumented function to load system libraries and print nicer error
+// messages.
+pub inline fn loadLibrary(libName: []const u8) !std.DynLib {
```

Comment:
Can you move this [here](https://github.com/hexops/mach/blob/main/src/main.zig#L37) instead and call it `dynLibOpen`?

Reply:
Updated!